extensions/cornerstone/src/utils/measurementServiceMappings/utils/selection.ts

```diff
@@ -0,0 +1,44 @@
+import { annotation as cs3dToolAnnotationUtils } from '@cornerstonejs/tools';
```

Comment:
The code in the 3 utils classes is used by the freehand mapping to centralize how the mapping is defined/handled.  Probably we will use these utilities further in the future.